interpreter.h

```diff
@@ -18,4 +29,4 @@ data_stack* interpreter_remove_active_stack(interpreter* interp);
 
 void interpreter_free(interpreter* interp);
 
-#endif//__interpreter_h_
+#endif //__interpreter_h_
```

Comment:
The lack of space was so that it lined up with the include guard statements at the start of the program.